src/main/java/frc/robot/Constants.java

```diff
@@ -14,8 +15,13 @@ public class Constants {
     public static final CANAddress SWERVE_DRIVE_LEFT_REAR = new CANAddress(3);
     public static final CANAddress SWERVE_DRIVE_RIGHT_FRONT = new CANAddress(1);
     public static final CANAddress SWERVE_DRIVE_RIGHT_REAR = new CANAddress(4);
+    public static final CANAddress ELEVATORA = new CANAddress(0);
+    public static final CANAddress ELEVATORB = new CANAddress(0);
+    public static final CANAddress ELEVATOR_WRIST = new CANAddress(0);
```

Comment:
These IDs are available in #electrical_team

```suggestion
    public static final CANAddress ELEVATORA = new CANAddress(4);
    public static final CANAddress ELEVATORB = new CANAddress(9);
    public static final CANAddress ELEVATOR_WRIST = new CANAddress(2);
```